internal_event_handlers.go

```diff
@@ -335,6 +335,40 @@ func (wc *workflowEnvironmentImpl) RequestCancelTimer(timerID string) {
 	wc.logger.Debug("RequestCancelTimer", zap.String(tagTimerID, timerID))
 }
 
+func validateVersion(changeID string, version, minSupported, maxSupported Version) {
```

Comment:
you can return error instead of flag. it is still good to have validation as a separate method.